### PR TITLE
major update to work safely on GS cluster

### PR DESCRIPTION
**Pull request by puja108**

### CODE REVIEW

**Comment by puja108** [comment]
need to wait till their image is released...

**Comment by puja108** [comment]
WFM, on minikube 0.11.0 (k8s 1.4.0)

RFR @webwurst

**Comment by puja108** [comment]
I took over this issue for syncing with current status of the clusters. Updates to both prom and grafana are included. There's also an added ingress for grafana, which now comes with basic auth enabled, docs need updating still.

**Comment by puja108** [comment]
Should be all done now. RFR @webwurst @marians @oponder 

It is all deployed on l8 cluster if you want to check. Ask me for the grafana password if you want to check that out.

File: docs/index.md [hunk 1]
```diff
@@ -15,87 +15,41 @@ It offers a lot of integrations incl. Docker, Kubernetes, etc.
...
-	- `Type`: `Prometheus`
-	- `Url`: `http://prometheus:9090`
-	- `Add`
+## Checking Grafana
 
-![Grafana Datasource](grafana_datasource.png)
+Now that we know Prometheus is up and running we can check for Grafana.
 
-Then go to the Dashboards tab and import the [Prometheus Stats dashboard](https://grafana.net/dashboards/2), which shows the status of Prometheus itself.
+There's an ingress set up for Grafana, so it should be available at https://http://grafana.monitoring.<cluster-id>.k8s.gigantic.io/
```
Comment (marians) [inline docs/index.md]: Typo: `https://http://`

Also, please put the URL in backticks.

File: docs/index.md [hunk 2]
```diff
@@ -15,87 +15,41 @@ It offers a lot of integrations incl. Docker, Kubernetes, etc.
...
 
-![Grafana Datasource Dashboard](grafana_datasource_dashboard.png)
+You can user the default admin (`admin:admin`) user for your first login. You should this admin user to reflect your desired username, your email, and a secure password ASAP!
 
-You can check it out to see how your Prometheus is doing.
+## Changing the admin
 
-![Grafana Datasource Dashboard](grafana_prometheus_stats.png)
+You can change the default admin user at http://grafana.monitoring.<cluster-id>.k8s.gigantic.io/admin/users/edit/1
```
Comment (marians) [inline docs/index.md]: As above, URL in backticks please

File: docs/index.md [hunk 3]
```diff
@@ -15,87 +15,41 @@ It offers a lot of integrations incl. Docker, Kubernetes, etc.
...
-
-Further, we need the Prometheus Node Exporter deployed to each node. For this we use a Daemon Set and a fronting service for Prometheus to be able to access the node exporters.
-
-```
-kubectl create --filename manifests/node-exporter-service.yaml
-kubectl create --filename manifests/node-exporter-daemonset.yaml
-```
-
-Wait a bit for all the pods to come up. Then Prometheus should be ready and running. We can check the Prometheus targets at https://mycluster.k8s.gigantic.io/api/v1/proxy/namespaces/default/services/prometheus/targets
+Wait a bit for all the pods to come up. Then Prometheus should be ready and running. We can check the Prometheus targets at https://<cluster-id>.k8s.gigantic.io/api/v1/proxy/namespaces/default/services/prometheus:9090/targets
```
Comment (marians) [inline docs/index.md]: We need backticks here around the URL, otherwise `<cluster-id>` won't show up.

File: docs/index.md [hunk 4]
```diff
@@ -15,87 +15,41 @@ It offers a lot of integrations incl. Docker, Kubernetes, etc.
...
-## Setting Up Grafana
-
-TLDR: If you don't want to go through all the manual steps below you can let the following job use the API to configure Grafana to a similar state.
-
-```bash
-kubectl create --filename manifests/grafana-import-dashboards-job.yaml
-```
-
-Once we're in Grafana we need to first configure [Prometheus](https://grafana.net/plugins/prometheus) as a data source.
+*Note*: The above URL uses your Kubernetes API to proxy to the service. As the API is guarded with your credentials, you need to [set them up in your system](/guides/accessing-services-from-the-outside/) (and/or browser). We do not recommend to set up an Ingress for Prometheus at this time, as it currently does not support any kind of authentification and thus your cluster would be open to everyone.
```
Comment (marians) [inline docs/index.md]: `authentification`? I'd use `authentication`. See http://english.stackexchange.com/questions/7844/is-authentification-a-real-word

File: docs/index.md (hunk 1 above)
Comment (marians) [inline docs/index.md]: Might want to check `ingress` vs. `Ingress` spelling consistency.

File: docs/index.md [hunk 5]
```diff
@@ -15,87 +15,41 @@ It offers a lot of integrations incl. Docker, Kubernetes, etc.
...
 
-Last, but not least we can import a sample [Kubernetes cluster monitoring dashboard](https://grafana.net/dashboards/162), to get a first overview over our cluster metrics.
+![Grafana Datasource](grafana_edit_admin.png)
 
--  `Grafana UI / Dashboards / Import`
-	- `Grafana.net Dashboard`: `https://grafana.net/dashboards/162`
-	- `Load`
-	- `Prometheus`: `prometheus`
-	- `Save & Open`
+Please note, that you need to update the Password and the user data (username, email, etc.) separately with the respective update buttons below each section.
```
Comment (marians) [inline docs/index.md]: Password > password

File: docs/index.md (hunk 4 above)
Comment (puja108) [inline docs/index.md]: ack

File: docs/index.md (hunk 1 above)
Comment (puja108) [inline docs/index.md]: thx

File: docs/index.md (hunk 3 above)
Comment (marians) [inline docs/index.md]: Still no backticks there around `https://<cluster-id>.k8s.gigantic.io/api/v1/proxy/namespaces/default/services/prometheus:9090/targets`
